synopsys-task/src/synopsys-task/model/polaris.ts

```diff
@@ -12,9 +15,15 @@ export interface PolarisData {
   branch: Branch;
   project: { name: string };
   assessment: { types: string[] };
+  prcomment?: PRComment;
```

Comment:
Although bridge accepts values case insensitive, but we can make it consistent.
Current bridge mapping is polaris.prComment.enabled so we can make it as prComment

Reply:
I believe, we are following the same pattern for other platforms as well in synopsys-extension, so we should be good.

Reply:
in action also we have prComment https://github.com/synopsys-sig/synopsys-action/blob/main/src/synopsys-action/input-data/polaris.ts#L17

Reply:
In action it might be. Extension codebase we are following prcomment.